packages/nx/src/utils/command-line-utils.ts

```diff
@@ -170,7 +170,8 @@ export function splitArgsIntoNxArgsAndOverrides(
   } else if (
     args['parallel'] === 'true' ||
     args['parallel'] === true ||
-    args['parallel'] === ''
+    args['parallel'] === '' ||
```

Comment:
What is your opinion about creating an array with all the available options and calling the `include()` function to check if `args['parallel']` is matching one of the array's items?

**what are the benefits in my opinion?**
- Readability (easy to understand what are the available options in one place)
- Maintainability (you can always add more cases by adding a new item to the array)